cms/src/components/content/EditContentBasic.vue

```diff
@@ -1,133 +1,47 @@
-        if (!content.value.slug) {
-            content.value.slug = Slug.generateNonUnique(content.value.title);
-        }
-
-        // Auto-update the slug if the title changes when in draft mode (unless the slug has been manually changed)
-        if (
-            titleChanged &&
-            content.value.status == PublishStatus.Draft &&
-            content.value.slug.replace(/-[0-9]*$/g, "") ==
-                Slug.generateNonUnique(previousTitle).replace(/-[0-9]*$/g, "")
-        ) {
-            // TODO: This sometimes creates a race condition
-            content.value.slug = Slug.generateNonUnique(content.value.title);
-        }
-
-        // Validate slug
-        if (slugChanged) {
-            content.value.slug = Slug.generateNonUnique(content.value.slug);
-        }
-
-        previousTitle = content.value.title;
-        previousSlug = content.value.slug;
-    },
-    { deep: true },
-);
+const content = defineModel<ContentDto>("content");
 
-const validateSlug = async () => {
-    if (!content.value) return;
-    content.value.slug = await Slug.generate(content.value.slug, content.value._id || "");
-};
+const activeTab = ref(0);
+const tabs = [
+    { title: "General ", component: ContentBasic },
+    { title: "SEO ", component: ContentSEO },
+];
 </script>
 
 <template>
-    <LCard title="Basic translation settings" collapsible v-if="content">
-        <!-- Title -->
-        <LInput
-            name="title"
-            label="Title"
-            required
-            :disabled="disabled"
-            v-model="content.title"
-            @blur="validateSlug"
-        />
-
-        <!-- Slug -->
-        <div class="mt-2 flex gap-1 align-top text-xs text-zinc-800">
-            <span class="py-0.5">Slug:</span>
-            <span
-                v-show="!isEditingSlug"
-                data-test="slugSpan"
-                class="inline-block rounded-md bg-zinc-200 px-1.5 py-0.5"
-                >{{ content.slug }}</span
-            >
-            <LInput
-                v-show="isEditingSlug"
-                :disabled="disabled"
-                ref="slugInput"
-                name="slug"
-                size="sm"
-                class="w-full"
-                v-model="content.slug"
-                @blur="
-                    isEditingSlug = false;
-                    validateSlug();
-                "
-            />
-            <button
-                data-test="editSlugButton"
-                v-if="!isEditingSlug && !disabled"
-                @click="startEditingSlug"
-                class="flex h-5 w-5 min-w-5 items-center justify-center rounded-md py-0.5 hover:bg-zinc-200 active:bg-zinc-300"
-                title="Edit slug"
-            >
-                <component :is="PencilIcon" class="h-4 w-4 text-zinc-500" />
-            </button>
+    <LCard title="General Settings" collapsible v-if="content">
+        <!-- Tab Navigation -->
+        <div>
+            <nav class="flex space-x-4">
```

Comment:
This is the correct place for the tabs instead of inside `LCard`, but just use the `LTabs` component for it (see my previous comment).